cmd/commento/main.go

```diff
@@ -15,23 +16,23 @@ func main() {
 	if err != nil {
 		Die(err)
 	}
-	
+
+	// Load configuration from the environment.
+	// Values in earlier files will take precedence over later values
```

Comment:
Earlier files do take precedence. If you have a .env.development.local, it is assumed that you want those values to be primary because you're developing.
Deployments would just contain only the .env they intend to use, like .env.production

However, if you do not set a value then later files will set the value. The first one to set wins.